### PR TITLE
Deprecate redundant properties

### PR DESCRIPTION
**Pull request by lmatas**

### CODE REVIEW

**Review by paulo-graca — CHANGES_REQUESTED** [review]
Based on my feedback here:  https://github.com/lareferencia/lareferencia-lrharvester-app/pull/32
this PR doesn't work as expected on my end.

Majorly, the validation report is empty and I don't get any errors on my end.

File: src/main/java/org/lareferencia/backend/workers/validator/ValidationWorker.java [hunk 1]
```diff
@@ -121,27 +121,29 @@ public void preRun() {
 			try {
 	
 				
-				if ( runningContext.getNetwork().getBooleanPropertyValue("VALIDATE") )
+				//if ( runningContext.getNetwork().getBooleanPropertyValue("VALIDATE") )
```
Comment (paulo-graca) [inline src/main/java/org/lareferencia/backend/workers/validator/ValidationWorker.java]: Will you left this here? Or it's just for testing?

File: src/main/java/org/lareferencia/backend/workers/validator/ValidationWorker.java [hunk 2]
```diff
@@ -121,27 +121,29 @@ public void preRun() {
...
 					
 					if ( runningContext.getNetwork().getValidator() != null ) {
 						validator = validationManager.createValidatorFromModel(runningContext.getNetwork().getValidator());
 					} else {
 						logInfo("No Validator was found for " + runningContext.toString() + "!!!");
 					}
 	
-				if (  runningContext.getNetwork().getBooleanPropertyValue("TRANSFORM") ) {
+				//if (  runningContext.getNetwork().getBooleanPropertyValue("TRANSFORM") ) {
```
Comment (paulo-graca) [inline src/main/java/org/lareferencia/backend/workers/validator/ValidationWorker.java]: Will you left this here? Or it's just for testing?

File: src/main/java/org/lareferencia/core/harvester/workers/HarvestingWorker.java [hunk 3]
```diff
@@ -152,7 +159,8 @@ public void run() {
 
 
 		// if fetch identify is true, will try to fetch the identify information
-		if ( runningContext.getNetwork().getBooleanPropertyValue("HARVEST_IDENTIFY_PARAMETERS") ) {
+		//if ( runningContext.getNetwork().getBooleanPropertyValue("HARVEST_IDENTIFY_PARAMETERS") ) {
```
Comment (paulo-graca) [inline src/main/java/org/lareferencia/core/harvester/workers/HarvestingWorker.java:162]: Will you left this here? Or it's just for testing?

Comment (lareferencia) [reply src/main/java/org/lareferencia/core/harvester/workers/HarvestingWorker.java:162]: Now it does not use the property anymore, I've just commented the line for the moment, We should remove it soon.
Now it uses this.fetchIdentifyParameters, which is a worker property instead.
Please let me know if I am missing something here.

**Comment by lmatas** [comment]
@paulo-graca Can we consider https://github.com/lareferencia/lareferencia-platform/issues/1 solved?

**Review by paulo-graca — DISMISSED** [review]
Thank you @lmatas. It works with most recent version !

**Comment by lmatas** [comment]
@paulo-graca Since this update will change the view/behaviour  of the properties for the actual users I think that we should:

1. Change the minor version, since is not a simple bugfix
2. Provide some documentation of the new behaviour ( we need to provide a changelog document for now on)

What do you think?

**Comment by paulo-graca** [comment]
My principle is whatever is easier to create and maintain and could help others in the long run.
For this particular case, I don't think we have interface tutorials, so, this disruption will not affect documentation. I agree that we should work based on releases as we tried in the past and every release should have a changelog (perhaps based on commits and issues)

**Comment by lmatas** [comment]
Thanks @paulo-graca I will work on a solution and let you know. Do you
think that we should promote this to master?
Lautaro Matas

**Pull request merged**